cmd/palomar/README.md

```diff
@@ -1,37 +1,33 @@
-# palomo
+# Palomar
 
-An elasticsearch/opensearch frontend and ATP repo crawler meant to provide
-search services for the bluesky network.
+Palomar is an Elasticsearch/OpenSearch frontend and ATP (ActivityPub) repository crawler designed to provide search services for the Bluesky network.
+
+## Prerequisites
+
+- GoLang (version 1.x)
+- Running instance of Elasticsearch or OpenSearch for indexing.
+- Valid credentials for the Personal Data Store (PDS) you want to index against.
 
 ## Building
 
 ```
 go build
 ```
 
-## Setup
-
-You will need a running elasticsearch instance (or cluster) for indexing, and
-valid credentials for the PDS you wish to index against.
+## Configuration
 
-The following environment variables should be set:
+Palomar uses environment variables for configuration. Here are the required ones:
```

Comment:
these are not all "required", only some of them are necessary. the `--help` on the CLI clarifies, I think.

Reply:
I do not see a help option, but perhaps I'm missing it.

Reply:
```
bnewbold@snocat:~/code/indigo$ go run ./cmd/palomar/ --help
NAME:
   palomar - search indexing and query service (using ES or OS)

USAGE:
   palomar [global options] command [command options] [arguments...]

VERSION:
   dev-

COMMANDS:
   elastic-check  
   search         run a simple query against search index
   run            combined indexing+query server
   help, h        Shows a list of commands or help for one command

GLOBAL OPTIONS:
   --elastic-cert-file value  certificate file path [$ES_CERT_FILE, $ELASTIC_CERT_FILE]
   --elastic-username value   elasticsearch username (default: "admin") [$ES_USERNAME, $ELASTIC_USERNAME]
   --elastic-password value   elasticsearch password (default: "admin") [$ES_PASSWORD, $ELASTIC_PASSWORD]
   --elastic-hosts value      elasticsearch hosts (schema/host/port) (default: "http://localhost:9200") [$ES_HOSTS, $ELASTIC_HOSTS]
   --es-post-index value      ES index for 'post' documents (default: "posts") [$ES_POST_INDEX]
   --es-profile-index value   ES index for 'profile' documents (default: "profiles") [$ES_PROFILE_INDEX]
   --bgs-host value           hostname and port of BGS to subscribe to (default: "https://bsky.social") [$ATP_BGS_HOST]
   --plc-host value           method, hostname, and port of PLC registry (default: "https://plc.directory") [$ATP_PLC_HOST]
   --pds-host value           method, hostname, and port of PDS instance (default: "https://bsky.social") [$ATP_PDS_HOST]
   --help, -h                 show help
   --version, -v              print the version
```